execution/state.go

```diff
@@ -172,6 +181,84 @@ func MakeGenesisState(db dbm.DB, genesisDoc *genesis.GenesisDoc) (*State, error)
 	return s, nil
 }
 
+func (s *State) LoadDump(filename string, genesisDoc *genesis.GenesisDoc) error {
+	f, err := os.OpenFile(filename, os.O_RDONLY, 0644)
+	if err != nil {
+		return err
+	}
+
+	tx := exec.TxExecution{
+		TxType: payload.TypeCall,
+		TxHash: make([]byte, 32),
+	}
+
+	for {
+		var row dump.Dump
+
+		_, err = cdc.UnmarshalBinaryLengthPrefixedReader(f, &row, 0)
+		if err != nil {
+			break
+		}
+
+		if row.Account != nil {
+			if row.Account.Address == crypto.ZeroAddress {
```

Comment:
ugh this isn't on this PR but I'd really like to refactor the global permissions address to something other than ZeroAddress, and ideally have a consistent way of thinking about/checking 'registered addresses'.

Could we for the time being introduce a constant in permissions package (which should be initially set to ZeroAddress) for the global permissions address to be clear about what this check is meant to be saying.

Reply:
Using acm.GlobalPermissionsAddress instead